pom.xml

```diff
@@ -26,13 +26,13 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.6</version>
+    <version>2.22</version>
   </parent>
 
   <artifactId>throttle-concurrents</artifactId>
   <packaging>hpi</packaging>
   <name>Jenkins Throttle Concurrent Builds Plug-in</name>
-  <version>1.9.1-SNAPSHOT</version>
+  <version>2.0-SNAPSHOT</version>
```

Comment:
I would advice going through the beta- or -rc for this change. The potential impact is too high.